### PR TITLE
SY-577 Documentation Links, SY-883 TypeScript README

### PR DESCRIPTION
**Pull request by pjdotson**
1. Updated links to the documentation website in the repository ([SY-577](https://linear.app/synnax/issue/SY-577/fix-in-code-documentation-links)).

2. Updated TypeScript Client `package.json`, `README.md`, and made a separate `CONTRIBUTING.md` file ([SY-883](https://linear.app/synnax/issue/SY-883/fix-typescript-client-readme)).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,19 +35,18 @@ scale.
...
 Our team is targeting a v1 release before the end of 2024, at which point all APIs
 will be stable and follow strict semantic versioning.
 
 # How to Contribute
 
-Help us modernize industrial control! Reach out
-to [Emiliano](mailto:ebonilla@synnaxlabs.com)(ebonilla@synnaxlabs.com) if you'd like to
-get involved. While you wait for a response, check out
-the [New Contributor Guide](docs/CONTRIBUTING.md) to get up to speed.
+Help us modernize industrial control! Reach out to [Emiliano](mailto:ebonilla@synnaxlabs.com)
```
Comment (emilbon99) [inline README.md:47]: We should honestly get rid of this.

Comment (emilbon99) [reply README.md:47]: We're not looking for contributors

File: client/ts/package.json [hunk 2]
```diff
@@ -15,6 +12,17 @@
     "telemetry",
     "control systems"
   ],
+  "homepage": "https://github.com/synnaxlabs/synnax/tree/main/client/ts",
+  "bugs": {
+    "url": "https://github.com/synnaxlabs/synnax/issues"
+  },
+  "license": "BUSL-1.1",
```
Comment (emilbon99) [inline client/ts/package.json:19]: Just `BSL` not `BUSL`

File: client/ts/README.md [hunk 3]
```diff
@@ -1,51 +1,36 @@
...
 
 ## Installation
 
 If you want to install the client library, please install it via a package manager such
 as npm or yarn:
 
-```shell
-npm install @synnaxlabs/client
+```bash
+npm i @synnaxlabs/client
```
Comment (emilbon99) [inline client/ts/README.md:19]: I generally prefer using the whole `install` command, as it seems clearer to someone new to javascript.

**Review by emilbon99 — APPROVED** [review]
Approved once you address my comments

File: client/ts/package.json (hunk 2 above)
Comment (pjdotson) [reply client/ts/package.json:19]: It is `BUSL-1.1` in the [SPDX license](https://spdx.org/licenses/BUSL-1.1.html) database. [BSL](https://spdx.org/licenses/BSL-1.0.html) is the Boost Software License.

**Review by Lham42 — APPROVED** [review]

**Pull request merged**